Makefile-test.mk

```diff
@@ -99,6 +94,14 @@ test-integration: gomod-download envtest ginkgo dep-crds kueuectl ginkgo-top ##
 	$(GINKGO) $(INTEGRATION_FILTERS) $(GINKGO_ARGS) $(GOFLAGS) -procs=$(INTEGRATION_NPROCS) --race --junit-report=junit.xml --json-report=integration.json --output-dir=$(ARTIFACTS) -v $(INTEGRATION_TARGET)
 	$(PROJECT_DIR)/bin/ginkgo-top -i $(ARTIFACTS)/integration.json > $(ARTIFACTS)/integration-top.yaml
 
+.PHONY: test-integration-required
+test-integration-required: INTEGRATION_FILTERS= --label-filter="!slow && !redundant"
+test-integration-required: test-integration ## Run required integration tests for singlecluster suites.
+
+.PHONY: test-integration-slow-and-redundant
+test-integration-slow-and-redundant: INTEGRATION_FILTERS= --label-filter="slow || redundant"
```

Comment:
I'm not sure about the naming `test-integration-slow-and-redundant`. Slow is fine, but the "redundant" sounds like we should not run them at all. Also `test-integration-required` might be misleading, because `test-integration-slow-and-redundant` is also required. 

I would suggest renames:
 - `test-integration-required`  ->   `test-integration` or `test-integration-baseline`
 - `test-integration-slow-and-redundant` ->  `test-integration-slow` or `test-integration-extended` 

No strong view, but I would probably be leaning probably to `test-integration-baseline` and  `test-integration-extended` . wdyt @tenzen-y @PBundyra ?